Sources/ScipioKit/Producer/BinaryExtractor.swift

```diff
@@ -8,14 +8,16 @@ struct BinaryExtractor {
     var fileSystem: any FileSystem
 
     @discardableResult
-    func extract(of binaryTarget: BinaryTarget) throws -> URL {
-
+    func extract(of binaryTarget: BinaryTarget, overwrite: Bool) throws -> URL {
         let sourcePath = binaryTarget.artifactPath
+        let frameworkName = "\(binaryTarget.c99name).xcframework"
         let fileName = sourcePath.basename
         let destinationPath = outputDirectory.appendingPathComponent(fileName)
-        if !fileSystem.exists(destinationPath) {
-            try fileSystem.copy(from: sourcePath.asURL, to: destinationPath)
+        if fileSystem.exists(destinationPath) && overwrite {
```

Comment:
How about using `if fileSystem.exists(destinationPath), overwrite {` to make it swifty?

Reply:
I prefer to use `&&` to declare conditions in the if statement without `let` binding.